blocks/editable/format-toolbar.js

```diff
@@ -151,17 +151,19 @@ class FormatToolbar extends Component {
 						className="editable-format-toolbar__link-modal"
 						style={ linkStyle }
 						onSubmit={ this.submitLink }>
+						<label className="screen-reader-text" htmlFor="editable-format-toolbar__link-input">{ __( 'URL' ) }</label>
 						<input
 							autoFocus
 							className="editable-format-toolbar__link-input"
+							id="editable-format-toolbar__link-input"
```

Comment:
@youknowriad 
I attempted to make use of withinstanceId, but this fails to work.

I imported the component, and loaded it in the render function as a parameter, but I wasn't able to. I'm not sure whether this ID is needed to be passed into the constructor, or somewhere else, but I'm not able to figure out how to do that.

This is where I attempted to do this:
https://github.com/xwp/gutenberg/commit/7aef0027a2361defb8e1011b990b8257956ec9ed

The error I get is:
format-toolbar.js:121 Uncaught TypeError: Cannot read property 'instanceId' of undefined

I also double-checked and the "editable-format-toolbar__link-modal" only shows up once at all time in the DOM. When trying to add a new link the previous link modal is removed from the DOM before adding the new modal, therefore an ID isn't required in this particular case.

I haven't committed the uuid in this PR, so the feature is still fully functional.